uber/site_sections/accounts.py

```diff
@@ -70,7 +70,7 @@ def login(self, session, message='', original_location=None, **params):
 
         if 'email' in params:
             try:
-                account = session.get_account_by_email(params['email'])
+                account = session.get_account_by_email(normalize_email(params['email']))
```

Comment:
If we're normalizing .'s, there's no reason to not normalize other special characters.  Valid email addresses can contain:

- uppercase and lowercase Latin letters A to Z and a to z; 
- digits 0 to 9;
- special characters !#$%&'*+-/=?^_`{|}~;
- dot ., provided that it is not the first or last character unless quoted, and provided also that it does not appear consecutively unless quoted (e.g. John..Doe@example.com is not allowed but "John..Doe"@example.com is allowed);
- space and "(),:;<>@[\] characters are allowed with restrictions (they are only allowed inside a quoted string, as described in the paragraph below, and in addition, a backslash or double-quote must be preceded by a backslash);
- comments are allowed with parentheses at either end of the local-part; e.g. john.smith(comment)@example.com and (comment)john.smith@example.com are both equivalent to john.smith@example.com.

My suggestion for search would be to strip all special characters (unless wrapped in quotes), strip all comments, and covert the address to lowercase.